src/config/app.config.ts

```diff
@@ -42,7 +44,7 @@ export default registerAs('app', (): Record<string, any> => {
     port: getIntConfig(process.env.HTTP_PORT, 3001),
     apiKey: process.env.API_KEY,
     logLevel: process.env.LOG_LEVEL || 'info',
-    redactLogs: process.env.REDACT_LOGS !== 'false',
-    singleLineLogFormat: process.env.SINGLE_LINE_LOG_FORMAT !== 'false',
+    redactLogs: getBooleanConfig(process.env.REDACT_LOGS, true),
+    singleLineLogFormat: getBooleanConfig(process.env.SINGLE_LINE_LOG_FORMAT, false),
```

Comment:
Why did we change the default for `singleLineLogFormat`?

Reply:
Ah -- Changing back